src/watch.ts

```diff
@@ -112,7 +112,9 @@ export async function watchConfig<
     try {
       config = await loadConfig(options);
     } catch (error) {
-      console.warn(`Failed to load config file at ${path}. Please check your config file for syntax errors.`);
+      console.warn(
+        `Failed to load config file at ${path}. Please check your config file for errors. (${error})`,
```

Comment:
can you take a screenshot of outout now

Reply:
![image](https://github.com/user-attachments/assets/62fd73c9-c635-43c6-b796-10b7b2346f75)

Currently its this
Do you know if an Error always logs the path itself, then we could make the error message shorter

Reply:
It might or might not. honestly i think addition of `()` wrapper and extra `Please check your config file for errors` is really not necessary and can be even misleading. 

For example if issue is because there is `import "foo"` in config file not being installed, actually right action for user is to install it not checking the config file.

Not sure what is not currently clear that we could make more clear...